packages/calcite-components/.stylelintrc.cjs

```diff
@@ -3,7 +3,10 @@
 // ⚠️ AUTO-GENERATED CODE - DO NOT EDIT
 const customFunctions = [
   "get-trailing-text-input-padding",
-  "scale-duration"
+  "medium-modular-scale",
```

Comment:
Per our conversation, I'll look at why these were added since `small/medium-modular-scale` functions are not defined in `dev` or in this PR (maybe from `node_modules`? 🤔).